content/2017/03/16/serializing-things.rst

```diff
@@ -0,0 +1,211 @@
+public: yes
+author: kit
+tags: [Celery, Django, Serialization]
```

Comment:
'Code' tag should be added based on our new fancy guidelines :) http://oddbird.net/styleguide/index.html#tags